techio/part03StepVerifier.md

```diff
@@ -72,14 +72,15 @@ so you should usually put a `expectSubscription()` after `.withVirtualTime()` if
 going to use `expectNoEvent` right after).
 
 ```Java
-StepVerifier.withVirtualtime(() -> Mono.delay(Duration.ofHours(3)))
+StepVerifier.withVirtualTime(() -> Mono.delay(Duration.ofHours(3)))
             .expectSubscription()
             .expectNoEvent(Duration.ofHours(2))
             .thenAwait(Duration.ofHours(1))
             .expectNextCount(1)
+            .expectComplete()
             .verify();
 ```
 
 Let's try that by making a fast test of our hour-long publisher:
 
-@[Virtual time]({"stubs": ["/src/main/java/io/pivotal/literx/Part03StepVerifier.java"], "command": "io.pivotal.literx.Part03StepVerifierTest#countWithVirtualTime", "layout":"aside"})
```

Comment:
there doesn't seem to be any real diff here 🤔

Reply:
OK, got it .